ballerina/api_qualityinforecord_srv/docs.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "The `ballerinax/sap.s4hana.api_qualityinforecord_srv` package offers APIs for seamless integration with the [Quality Info Record API v1.2.0](https://api.sap.com/api/API_QUALITYINFORECORD_SRV/overview). The data related to a quality info record and first article inspection has one entity type each. For read operations, filter information must be sent according to the OData protocol.",
+  "communication-scenario": "Quality Info Record Integration",
+  "import-statement": "import ballerinax/sap.s4hana.api_qualityinforecord_srv as qltyinforecd;",
+  "client-init": "qltyinforecd:Client qltyInfoRecClient = check new (",
+  "api-invocation": "qltyinforecd:Wrapper CollectionOfQualityFirstArticleInspectionWrapper = check qltyInfoRecClient->listQltyFirstArticleInspsOfQualityInProcurement();"
```

Comment:
This this a valid api call? Seeems like return type is wrong. Also variable names should start in simple case

Reply:
https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/blob/f2c3a420df4cb0814fabff47a6db725adab9c26e/ballerina/api_qualityinforecord_srv/client.bal#L192

Your client says the return type is `CollectionOfQualityFirstArticleInspectionWrapper` not `Wrapper`